jenkins/github_helpers.py

```diff
@@ -23,6 +23,7 @@ def __init__(self):
         self._set_user_email()
         self._set_github_instance()
 
+    # FIXME: Does nothing, sets variable to None if env var missing
     def _set_github_token(self):
         try:
             self.github_token = os.environ.get('GITHUB_TOKEN')
```

Comment:
In place of the comments, could we just change to?:
```
self.github_token = os.environ['GITHUB_TOKEN']
```

Reply:
I'd like to, but there's a bit of a rathole here regarding tests. I discovered the no-op during manual testing, but it turned out to be pretty far out of scope.